dap-python.el

```diff
@@ -88,7 +88,7 @@ https://github.com/pyenv/pyenv-which-ext."
 (lsp-defun dap-python--parse-lsp-symbol
   ((&DocumentSymbol
     :name :kind
-    :selection-range (&Range :start (&Position :line start-line
+    :range (&Range :start (&Position :line start-line
```

Comment:
`selection-range` only have `start`-`end` range for `selection`, which cover the name of class/method.
`range` is what we need.